src/main/java/org/openhab/tools/analysis/checkstyle/ExportInternalPackageCheck.java

```diff
@@ -24,29 +24,29 @@
  */
 public class ExportInternalPackageCheck extends AbstractStaticCheck {
 
-    public final static String MANIFEST_EXTENSTION = "MF";
+    public final static String MANIFEST_EXTENSION = "MF";
```

Comment:
I guess that you have looked into this class to write your new check - and thus you have applied some formatting on it. Can you please  revert it back to make the PR clear.